server/src/main/clients/web-browser/src/ProjectPost/ProjectPost.tsx

```diff
@@ -0,0 +1,26 @@
+import {ReactNode} from 'react';
+
+function ProjectPost({
```

Comment:
Can you replace the component definition with the following? It doesn't require us to list out the parameter names twice:

function ProjectPost(props: {
  title: string;
  date: Date;
  image: string;
  content: ReactNode;
}) { ...

It does mean that we have to refer to "props." something. But, I think that is okay. Later, if we make things editable, you would do something like:

const [title, setTitle] = useState(props.title)

And, then you can edit "title" as you see fit. But, without props, the variable is already blocked by the parameter name.

Reply:
Done